src/java.base/share/classes/java/net/URL.java

```diff
@@ -809,6 +809,29 @@ void set(String protocol, String host, int port,
         }
     }
 
+    /**
+     * @return Returns the address of the host represented by this URL.
+     *         A {@link SecurityException} or an {@link UnknownHostException}
+     *         while getting the host address will result in this method returning
+     *         {@code null}
+     */
```

Comment:
The fix looks okay but the javadoc might be a big clearer if you move the text into the method description.

Reply:
Hello Alan, I've now updated the PR to move this text into the method description.